charts/metrics-server/templates/deployment.yaml

```
@@ -63,10 +63,14 @@ spec:
           - name: https
             protocol: TCP
             containerPort: {{ .Values.containerPort }}
+          {{- if .Values.livenessProbe }}
           livenessProbe:
             {{- toYaml .Values.livenessProbe | nindent 12 }}
+          {{- end }}
```

```suggestion
          {{- with .Values.livenessProbe }}
          livenessProbe:
            {{- toYaml . | nindent 12 }}
          {{- end }}
```

Could you use the `with` pattern here.